solidity/dashboard/src/components/AddETHModal.jsx

```diff
@@ -53,49 +50,14 @@ const AddEthModal = ({ operatorAddress, closeModal }) => {
         onSubmit={onSubmit}
         yourAddress={yourAddress}
         closeModal={closeModal}
+        action="add"
```

Comment:
I think this prop is unnecessary.

Reply:
ah I see.... Maybe `submitBtnText` instead of `action`?

Reply:
Also according to the Figma view, The `eth` word is missing here.

Reply:
yup, renamed and added "eth"